aggregator/elem_base.go

```diff
@@ -74,6 +74,16 @@ type createAggregationOptions struct {
 	initSourceSet bool
 }
 
+// IDMutationType configs if the id should be mutated after aggregation.
+type IDMutationType int
```

Comment:
Could you leave a comment in the diff description on what is the situation when we would need to add a prefix to a metric?

Reply:
Yeah we add a prefix and suffix when its a untimed aggregation, let me know if the description is not clear